assignments/prototypes.js

```diff
@@ -1,110 +1,145 @@
 /*
-  Object oriented design is commonly used in video games.  For this part of the assignment you will be implementing several constructor functions with their correct inheritance heirarchy.
+  Object oriented design is commonly used in video games.  For this part of the assignment
+  you will be implementing several classes with their correct inheritance heirarchy.
 
-  In this file you will be creating three constructor functions: GameObject, CharacterStats, Humanoid.  
-
-  At the bottom of this file are 3 objects that all end up inheriting from Humanoid.  Use the objects at the bottom of the page to test your constructor functions.
+  In this file you will be creating three classes: GameObject, CharacterStats, Humanoid.  
+  At the bottom of this file are 3 objects that all inherit from Humanoid.  Use the objects at the bottom of the page to test your classes.
   
-  Each constructor function has unique properites and methods that are defined in their block comments below:
+  Each class has unique properites and methods that are defined in their block comments below:
 */
   
 /*
   === GameObject ===
   * createdAt
   * dimensions
-  * destroy() // prototype method -> returns the string: 'Object was removed from the game.'
+  * destroy() // prototype method -> returns the string 'name was removed from the game.'
 */
+function GameObject(options) {
+  this.createdAt = options.createdAt;
+  this.dimensions = options.dimensions;
+}
 
-/*
-  === CharacterStats ===
-  * hp
-  * name
-  * takeDamage() // prototype method -> returns the string '<object name> took damage.'
-  * should inherit destroy() from GameObject's prototype
-*/
+GameObject.prototype.destroy = function () {
+  return `Object was removed from the game.`;
+};
```

Comment:
So this is actually supposed to reference the name of the object, not the word "Object" use something like `this.name` instead.